docs/solr.txt

```diff
@@ -36,102 +50,124 @@ index.search.solr.zookeeper-url=localhost:2181
 
-===== Connecting via HTTP
 
-When connecting to Solr via HTTP by setting the `mode` equal to `http` a single or list of URLs for the Solr instances must be provided.
+==== Configset Configuration
 
-[source, properties]
-----
-index.search.backend=solr
-index.search.solr.mode=http
-index.search.solr.http-urls=http://localhost:8983/solr
-----
+A configset is required to create a collection. The configset is stored in Zookeeper to enable access to it across the Solr servers.
 
-Additional configuration options for controlling the maximum number of connections, connection timeout and transmission compression are available for the HTTP mode. Refer to the <<config-ref>> for a complete listing of those options.
+* Each collection can provide its own configset when it is created, so that each collection may have a different configuration. With this approach, each collection must be created manually.
+* A shared configset can be uploaded separately to Zookeeper if it will be reused by multiple collections. With this approach, JanusGraph can create collections automatically by using the shared configset. Another benefit is that reusing a configset significantly reduces the amount of data stored in Zookeeper.
 
-==== Solr Collections
 
-.Note:
-[NOTE]
-Solr documentation makes a distinction between the terms *core* and *configuration set*.  For the purposes of configuring JanusGraph to work with Solr, the terms are used interchangeably.  
+===== Using an Individual Configset
 
-===== Collection Initialization
+In this example, a collection named `verticesByAge` is created manually using the default JanusGraph configuration for Solr that is found in the distribution. When the collection is created, the configuration is uploaded into Zookeeper, using the same collection name `verticesByAge` for the configset name. Refer to the https://lucene.apache.org/solr/guide/6_6/solr-control-script-reference.html#SolrControlScriptReference-CollectionsandCores[Solr Reference Guide] for available parameters.
 
-SolrCloud leverages Zookeeper to coordinate Solr *configuration sets* and *collection* information between instances of Solr.  Solr HTTP, because it is a single instance, keeps *configuration set* information exclusively on the file system. The use of Zookeeper with SolrCloud provides the opportunity to significantly reduce the amount of manual configuration required to use Solr as a back end index for JanusGraph.
+[source, bourne]
+----
+# create the collection
+$SOLR_HOME/bin/solr create -c verticesByAge -d $JANUSGRAPH_HOME/conf/solr
+----
 
+Define a mixed index using `JanusGraphManagement` and the same collection name.
 
+[source, gremlin]
+----
+JanusGraphManagement mgmt = graph.openManagement();
```

Comment:
This is marked as "gremlin" syntax (which I think makes sense) so drop type declarations (`JanusGraphManagement`, `PropertyKey`) and semi-colons.

Same comment for other Gremlin/Groovy blocks in this section.